File: microservicioEnvioCorreos - plantilla/src/main/java/co/edu/unicauca/infoii/correo/DTOs/CancionAlmacenarDTOInput.java
package co.edu.unicauca.infoii.correo.DTOs;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class CancionAlmacenarDTOInput {
    private String id;
    private String titulo;
    private String artista;
    private String genero;
}
	

File: servidorCanciones/plantilla/capaFachadaServices/DTOs/CancionAlmacenarDTOInput.go
package dtos

type CancionAlmacenarDTOInput struct {
	Titulo      string `json:"titulo"`
	Artista     string `json:"artista"`
	Album       string `json:"album"`
	Genero      string `json:"genero"`
	ReleaseYear int    `json:"release_year"`
	Idioma      string `json:"idioma"`
}
